tests/hdb_connector/dbapi_connector_test.py

```diff
@@ -101,7 +101,7 @@ def test_query(self, mock_logger):
         self._conn._connection = mock.Mock()
         self._conn._connection.cursor.return_value = context_manager_mock
 
-        response = self._conn.query('query')
+        meta_data, response = self._conn.query('query')
```

Comment:
These checks are quite fragile. I would add the check to confirm that the description matches. Its quite easy just add this:
```
cursor_mock_instance.description = "description:
```
```
self.assertEqual(meta_data, "description")
```